### PR TITLE
[MIG] Migrate account_invoice_zero

### PR DESCRIPTION
**Pull request by charbeljc**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please extract the description to a README.rst with the OCA template. You can add also the OCA generic icon, and provide some stylistic guides like directory layout, CamelCase and so on.

**Comment by alexis-via** [comment]
Code review, no test :+1:

File: account_invoice_zero/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Invoice Zero
+===========
```
Comment (tafaRU) [inline account_invoice_zero/README.rst]: Please remove 2 lines above

File: account_invoice_zero/README.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+receivable/payable move line with a 0 balance.  The invoice stays as
+open even if there is nothing to pay.  The user has 2 ways to set the
+invoice as paid: create a payment of 0 and reconcile the line with the
+payment or reconcile the receivable/payable move line with itself.
+This module takes the latter approach and will directly set the invoice
+as paid once it is opened.
+
+ 
+
+Installation
```
Comment (pedrobaeza) [inline account_invoice_zero/README.rst]: Remove or complete these useless sections

File: account_invoice_zero/README.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Firstname Lastname <email.address@example.org>
```
Comment (tafaRU) [inline account_invoice_zero/README.rst]: Please replace by using your credentials

**Comment by nbessi** [comment]
needs rebase

**Comment by charbeljc** [comment]
@nbessi rebase done, waiting for travis status.

**Comment by nbessi** [comment]
@charbeljc thanks :smile:

**Comment by vrenaville** [comment]
:+1:

File: account_invoice_zero/README.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+
+Account Invoice Zero
+====================
+
+Invoices with a amount of 0 are automatically set as paid.
+
+When an invoice has an amount of 0, OpenERP still generates a
```
Comment (pedrobaeza) [inline account_invoice_zero/README.rst]: s/OpenERP/Odoo

File: account_invoice_zero/README.rst [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+receivable/payable move line with a 0 balance.  The invoice stays as
+open even if there is nothing to pay.  The user has 2 ways to set the
+invoice as paid: create a payment of 0 and reconcile the line with the
+payment or reconcile the receivable/payable move line with itself.
+This module takes the latter approach and will directly set the invoice
+as paid once it is opened.
+
+ 
+
+For further information, please visit:
```
Comment (pedrobaeza) [inline account_invoice_zero/README.rst]: I would remove this useless sentence

File: account_invoice_zero/README.rst [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline account_invoice_zero/README.rst]: Remove empty section

**Comment by pedrobaeza** [comment]
The module name is not very instructive, so I would rename it to _account_invoice_zero_autopay_

**Comment by mdietrichc2c** [comment]
LGTM, :+1:

**Comment by gurneyalex** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Please take into account the module rename before merging

**Comment by damdam-s** [comment]
rename done

**Comment by pedrobaeza** [comment]
Merging then. Thanks!

**Pull request merged**